### PR TITLE
feat: featured package UI

### PR DESCRIPTION
**Pull request by fandihungryhub**
https://3.basecamp.com/5190892/buckets/31702802/card_tables/cards/5980740588

### CODE REVIEW

File: components/restaurant_detail/FeaturedPackageImage.vue [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<template>
+  <div class="relative mb-3">
+    <div class="featured-gradient absolute w-full h-full border-[5px] border-red-dark rounded-3xl">
+      <div class="flex flex-col absolute bottom-2.5 left-[15px]">
+        <span class="text-2xl font-bold text-white">Featured Package</span>
+        <span class="text-xs text-white">{{ props.packageType }}</span>
+      </div>
+    </div>
+    <img :src="props.imageSource" class="w-full rounded-3xl" />
```
Comment (afif-hh) [inline components/restaurant_detail/FeaturedPackageImage.vue:9]: pakai HhImage

**Review by afif-hh — CHANGES_REQUESTED** [review]

File: components/restaurant_detail/FeaturedPackage.vue [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import FeaturedPackageImage from "~~/components/restaurant_detail/FeaturedPackageImage.vue";
+import IcBaselinePerson from "~icons/ic/baseline-person";
+import IcSpoonFork from "~icons/icon-park-solid/fork-spoon";
+
```
Comment (afif-hh) [inline components/restaurant_detail/FeaturedPackage.vue:23]: kurang `defineOptions({ name: 'component name nya' })`

File: components/restaurant_detail/FeaturedPackageImage.vue (hunk 1 above)
Comment (afif-hh) [reply components/restaurant_detail/FeaturedPackageImage.vue:9]: kurang defineOptions juga

Comment (fandihungryhub) [reply components/restaurant_detail/FeaturedPackageImage.vue]: updated

File: components/restaurant_detail/FeaturedPackage.vue (hunk 2 above)
Comment (fandihungryhub) [reply components/restaurant_detail/FeaturedPackage.vue:23]: updated

**Review by afif-hh — APPROVED** [review]

**Pull request merged**